style.css

```diff
@@ -1,340 +1,926 @@
+    background-position: top left, bottom right;
+    transition: all 2s ease;
+}
+
+.catalog__item--hover {
+    background-color: #1D1C28;
+}
+
+.catalog__image {
+    position: relative;
+}
+
+.catalog__price {
+    font-family: Bakbak One, Arial, Helvetica, sans-serif;
+    font-size: 36px;
+    line-height: 48px;
+    letter-spacing: -0.5px;
+}
+
+.catalog__name {
+    font-size: 20px;
+    font-weight: 500;
+    line-height: 28px;
+    color: #EFF5FA;
+}
+
+/* Index page Blog block */
+
+.blog__wrapper {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+}
+
+.blog__post {
+    display: grid;
+    grid-row: span 2;
+    padding: 30px;
+    margin-right: 30px;
+    margin-bottom: 30px;
+    background-color: #1D1C28;
+}
+
+.blog__image {
+    position: relative;
+}
+
+.blog__image img {
+    width: 100%;
+    object-fit: cover;
+}
+
+.blog__post--small {
+    height: 435px;
+}
+
+.blog__post--small .blog__image {
+    overflow: hidden;
+}
+
+.blog__info {
+    display: flex;
+}
+
+.blog__name,
+.blog__date {
+    position: relative;
+    padding-left: 20px;
+}
+
+.blog__name {
+    padding-right: 36px;
+}
+
+.blog__name::before {
+    position: absolute;
+    content: '';
+    background-image: url('images/user.svg');
+    background-size: cover;
+    background-repeat: no-repeat;
+    top: 3px;
+    left: 0px;
+    width: 16px;
+    height: 16px;
+}
+
+.blog__name:after {
+    content: '';
+    position: absolute;
+    top: 4px;
+    right: 18px;
+    width: 2px;
+    height: 16px;
+    background-color: #D0D7FB2B;
+}
+
+.blog__date::before {
+    position: absolute;
+    content: '';
+    background-image: url('images/calendar.svg');
+    background-size: cover;
+    background-repeat: no-repeat;
+    top: 3px;
+    left: 0px;
+    width: 16px;
+    height: 16px;;
+}
+
+/* Index page Our Team block */
+
+.team {
+    max-width: 1300px;
+    margin-inline: auto;
+}
+
+.team__members {
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+}
+
+.team__item {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 316px;
+    height: 293px;
+    margin: 0 30px 30px 0;
+    padding: 30px;
+    background-color: #1D1C28;
+}
+
+.team__image {
+    display: block;
+    border-radius: 50%;
+}
+
+.team__name,
+.team__role {
+    text-align: center;
+}
+
+/* Index page About block */
+
+.about__wrapper {
+    max-width: 960px;
+    margin: 0 auto;
+}
+
+.about__title {
+    position: relative;
+    margin-top: 55px;
+    font-family: Bakbak One, Arial, Helvetica, sans-serif;
+    font-size: 24px;
+    line-height: 32px;
+    color: #fff;
+    z-index: 2;
+}
+
+.about__text {
+    font-size: 18px;
+    line-height: 28px;
+    color: #fff;
+}
+
+.about__features {
+    display: flex;
+    flex-wrap: wrap;
+    counter-reset: tidbit-counter;
+    list-style-type: decimal-leading-zero;
+}
+
+.about__features li:before {
+    position: absolute;
```

Comment:
If you use position: absolute, should be top (or  bottom), left (or right) values setted also.

Reply:
its was just a example, sorry. done